<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>313</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>313</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>313</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Редактирование</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="2">
    <widget class="QLineEdit" name="lineEdit_gate"/>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Гейт</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QComboBox" name="comboBox_status"/>
   </item>
   <item row="6" column="2">
    <widget class="QDateTimeEdit" name="dateTimeEdit_arrival_time"/>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBox_airline"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Статус</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Откуда</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Самолёт</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="comboBox_aircraft_type"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Куда</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="comboBox_departure"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Время прибытия</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDateTimeEdit" name="dateTimeEdit_departure_time"/>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="comboBox_arrival"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Время вылета</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Авиакомпания</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEdit_flight"/>
   </item>
   <item row="10" column="2">
    <widget class="QPushButton" name="cancel_button">
     <property name="text">
      <string>Отменить</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/cancel.png</normaloff>../icons/cancel.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>№ Рейса</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QPushButton" name="save_button">
     <property name="text">
      <string>Сохранить</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/save.png</normaloff>../icons/save.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
